lustre/changes/debian/rules

```diff
@@ -25,7 +25,7 @@ DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 config.status: configure
 	dh_testdir
 	# Add here commands to configure the package.
-	./configure  --enable-mpitests=no --with-linux=/vagrant/lustre/source/linux-2.6.32-431.17.1.el6.x86_64/
+	./configure  --enable-mpitests=no --with-linux=/mnt/vagrant/lustre/source/linux-2.6.32-573.3.1.el6.x86_64/
```

Comment:
/mnt/vagrant/lustre/source is just the place where the patched kernel is. Is there anything particular in AWS blocking us to create directories there?